<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDistributorSetting</class>
 <widget class="QWidget" name="FormDistributorSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>901</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>161</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_servo_on">
      <property name="text">
       <string>状态</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_servo_on">
      <property name="text">
       <string>伺服ON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_servo_off">
      <property name="text">
       <string>伺服OFF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_servo_error">
      <property name="text">
       <string>错误</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_ack_error">
      <property name="text">
       <string>报警确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_speedY">
      <property name="text">
       <string>Y速率</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_speedY"/>
    </item>
    <item>
     <widget class="QPushButton" name="button_speedY">
      <property name="text">
       <string>Y速率（mm/s）</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_posX">
      <property name="text">
       <string>X</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_moveX"/>
    </item>
    <item>
     <widget class="QPushButton" name="button_moveX">
      <property name="text">
       <string>X移位</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_posY">
      <property name="text">
       <string>Y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_moveY"/>
    </item>
    <item>
     <widget class="QPushButton" name="button_moveY">
      <property name="text">
       <string>Y移位</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
